tests/test_other.py

```diff
@@ -5726,8 +5726,9 @@ def test_failing_alloc(self):
           if not aborting: args += ['-s', 'ABORTING_MALLOC=0']
           print(args, pre_fail)
           check_execute(args)
-          output = run_js('a.out.js', stderr=PIPE, full_output=True, assert_returncode=None)
-          if (not aborting) or growth: # growth also disables aborting
+          manage_malloc = (not aborting) or growth # growth also disables aborting
```

Comment:
i'm not sure what "growth also disables aborting" has meant here. i'm probably forgetting something, but i think growth means we let the VM grow memory, and the test will allocate until the system runs out of memory. and the process would exit with a bad return code in that case, i think?

The test passes with your change, though, so I'm missing something.

Reply:
Oh, it will grow until the VM says it can't, right. Sounds good.